recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.6" %}
+{% set sha256 = "7ac29a8664b3708fc7909687fbb5d73125b0807139f2c76d4f7f2ded0f121a87" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    fn: py_gd-{{ version }}.tar.gz
+    url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    # git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    # git_tag: v{{ version }}
+
+build:
+    skip: True  # [py3k or win]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - libgd {{ libgd_version }}
+        - cython
+        - numpy x.x
+
+    run:
+        - python
+        - libgd {{ libgd_version }}
+        - numpy x.x
+
+test:
+    requires:
+        - pytest
+    imports:
+        - py_gd
+    commands:
+        - py.test --pyargs py_gd 
+
+about:
+    home: https://github.com/NOAA-ORR-ERD/py_gd
+    license: Public Domain
+    summary: 'Pythonic wrappers for the libgd graphics rendering library'
+
+extra:
+    recipe-maintainers:
+        # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+        - ChrisBarker-NOAA
+        - jay-hennen
```

Comment:
Also, as this will eventually come up, we keep track of general maintainer preferences in this issue ( https://github.com/conda-forge/staged-recipes/issues/139 ). If there are people that you are going to put down as maintainers on a lot of recipes, that is the place that they should go to so they can say what things they are generally ok with being added to.